internal/format/pretty.go

```diff
@@ -20,6 +20,17 @@ const (
 	{{ end -}}
 	`
 
+	prettyResourcesTpl = `
+	{{- if .Settings.ShowResources -}}
+		{{- with .Module.Resources }}
+			{{- range . }}
+				{{- printf "resource.%s" .FullType | colorize "\033[36m" }}
```

Comment:
I think it'd be nice to show the URL in parentheses here (no color), so one can just CTRL+click it from their terminal.